app/src/main/java/io/left/meshim/controllers/RightMeshController.java

```diff
@@ -48,6 +49,7 @@
     private AndroidMeshManager meshManager = null;
 
     // Set to keep track of peers connected to the mesh.
+    private HashSet<MeshID> discovered = new HashSet<>();
```

Comment:
What is the purpose of this list?

Reply:
This list keeps track of which meshids the discovery thing has been sent to, otherwise it will send on every update. Once its been sent, the meshid is added to the list. When the device leaves it will be removed. It handles the weird behaviour or having sometimes an ADDED or sometimes an UPDATED.

Reply:
Oh sure, it's right there. :upside_down_face: I am just _killing_ this code review thing today.